source/platforms/_tests/_github.test.ts

```diff
@@ -34,6 +34,10 @@ jest.mock("../github/GitHubAPI", () => {
       const fixtures = await requestWithFixturedJSON("requested_reviewers.json")
       return await fixtures()
     }
+
+    getExternalAPI() {
```

Comment:
@orta since our tests mock the `GitHubAPI` file, we need to supply the `getExternalAPI()` function here for the tests to pass. 👍